### PR TITLE
Added Travis CI testing

### PR DESCRIPTION
**Pull request by bheptinh**
**Note:** Branch was created from [1.x.x-starter](https://github.com/lemberg/draft-theme/tree/1.x.x-starter)

### CODE REVIEW

File: STARTER/.gitignore [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Node modules #
+node_modules
+bower_components
+
+# Numerous always-ignore extensions #
+*.sass-cache
+*.map
```
Comment (Crazylord) [inline STARTER/.gitignore]: Add empty space

File: STARTER/.jshintignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+node_modules/**
+bower_components/**
```
Comment (Crazylord) [inline STARTER/.jshintignore]: Add empty space

File: STARTER/STARTER.info.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+name: STARTER
+type: theme
+description: 'A subtheme of Classy, built for STARTER'
```
Comment (Crazylord) [inline STARTER/STARTER.info.yml]: Draft theme are base for this theme

File: STARTER/STARTER.info.yml [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+core: 8.x
+# Defines the base theme
+base theme: classy
```
Comment (Crazylord) [inline STARTER/STARTER.info.yml]: Draft theme are base for this theme

File: STARTER/gulpfile.js [hunk 5]
```diff
@@ -0,0 +1,285 @@
...
+    )
+    .pipe(autoprefixer(autoprefixerVersions))
+    .pipe(sourcemaps.write())
+    .pipe(concat('style.css'))
+    .pipe(gulp.dest('./css/'))
+    .pipe(cleanCss({
+      compatibility: 'ie11'
+    }))
+    .pipe(rename({
+      extname: '.min.css'
```
Comment (Crazylord) [inline STARTER/gulpfile.js]: Don't need *.min.css

**Comment by bheptinh** [comment]
@Crazylord seems like all comments are related PR #16

**Review by T2L — APPROVED** [review]

**Pull request merged**